qaframework-bdd-tests/src/test/resources/features/refapp-2.x/provider/provider.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Provider Management
+
+  Background: 
+    Given User clicks the system administration link on the home page
```

Comment:
what happens when a user clicks on the system administration link?

Reply:
he is redirected to the system administration page where the test clicks the advanced administration link in order to go to the advanced administration page

Reply:
which page is loaded when the user clicks on the system administration link?

Reply:
does this function answer you?
```
	public AdministrationPage goToAdministration() {
		clickOn(SYSTEM_ADMINISTRATION);
		return new SystemAdministrationPage(this).goToAdvancedAdministration();
	}
```

otherwise,

Reply:
let me phrase the question differently, do we have a gherkin step that asserts the system loads the System Administration page is successfully?

Reply:
actually i would have to assert the administration page instead because when you look at the function i shared [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/316#discussion_r785481183), it's a 2-in-1 kind-of. it combines two steps, instead of having `And User clicks on the system admin link` and `And User clicks on advanced admin link `, that function combines them

Reply:
actually i would have to assert the administration page instead because when you look at the function i shared [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/316#discussion_r785481183), it's a 2-in-1 kind-of. it combines two steps, instead of having `And User clicks on the system admin link` and `And User clicks on advanced admin link `, that function combines them

Reply:
@mherman22 do you mind taking a look at the existing feature files?